org.eclipse.paho.client.mqttv3/src/main/java/org/eclipse/paho/client/mqttv3/MqttConnectOptions.java

```diff
@@ -81,7 +81,7 @@
 	private String[] serverURIs = null;
 	private int mqttVersion = MQTT_VERSION_DEFAULT;
 	private boolean automaticReconnect = false;
-
+	private Properties customHeaders = null;
```

Comment:
Is there a reason to use the type `Properties` instead of a more generic `Map<String,String>`? `Properties` implement the very old and obsolete `Dictionary` interface and new code should not necessarily use it if no old java properties file is read.

Reply:
I can see that Properties used for sslClientProps
```java
 private Properties sslClientProps = null;
```
From the beginning I used Map, but changed to Properties for uniformity with existed sslClientProps.